src/doc/rustdoc/src/the-doc-attribute.md

```diff
@@ -214,3 +214,44 @@ the `strip-hidden` pass is removed.
 Since primitive types are defined in the compiler, there's no place to attach documentation
 attributes. This attribute is used by the standard library to provide a way to generate
 documentation for primitive types.
+
+## `#[doc(cfg(...))]` / `#[cfg(rustdoc)]`: Documenting platform-/feature-specific information
+
+Because of the way Rustdoc documents a crate, the documentation it creates is specific to the target
+rustc compiles for. Anything that's specific to any other target is dropped via `#[cfg]` attribute
+processing early in the compilation process. However, Rustdoc has a trick up its sleeve to handle
+platform-specific code if it *does* receive it.
+
+Because Rustdoc doesn't need to fully compile a crate to binary, it replaces function bodies with
```

Comment:
Unsure if this implementation detail needs to be exposed.

Reply:
I found it interesting so I didn't remove it. :)